### PR TITLE
Revamp FAQ and Transfer ROSE to ParaTimes chapters

### LINKED ISSUE #637: Update transfer Rose page to emphasize Sapphire
#### Description

Update this page to emphasize Sapphire.
https://docs.oasis.io/general/manage-tokens/how-to-transfer-rose-into-paratime/

Change text to Sapphire instead.
https://github.com/oasisprotocol/docs/blob/30922bcceaa02a77425d50b94683f61dd292afc4/docs/general/manage-tokens/how-to-transfer-rose-into-paratime.mdx#L152-L153

### PR DESCRIPTION
**Pull request by matevz**
This PR:
- Revamps the FAQ to include unmnemonic usage for Ledger and BitPie users. - [Preview](https://deploy-preview-651--trusting-archimedes-14c863.netlify.app/general/manage-tokens/faq#i-lost-my-ledger-or-my-ledger-is-broken-i-urgently-need-to-access-my-assets-can-i-import-ledger-mnemonic-into-oasis-wallet)
- FAQ now uses fancy tabs for OS selection and was renamed to faq.mdx
- bumps `external/cli` for new `oasis wallet import-file` example and `external/adrs` for faq.md -> faq.mdx link fix
- Change ROSE depositing chapter primarily for Sapphire + new screenshots (fixes #637) - [Preview](https://deploy-preview-651--trusting-archimedes-14c863.netlify.app/general/manage-tokens/how-to-transfer-rose-into-paratime)
- Cosmetic fixes (ADR 0008 -> ADR-8)

### CODE REVIEW

File: docs/general/manage-tokens/faq.md [hunk 1]
```diff
@@ -7,18 +7,13 @@ party wallets & custody providers supporting ROSE tokens.
...
 
 :::
 
-As of May 2023, the newer versions of BitPie wallet [do not support transferring
-tokens on the Oasis network anymore][bitpie-announcement]. Previously, sending
-tokens from the BitPie wallet to the address generated by the Oasis wallet was
-the preferred migration procedure.
-
-Thus, the only way to access your tokens stored on your BitPie wallet is to
+In May 2023, the BitPie team [removed support for the Oasis network][bitpie-announcement]. Thus, the only way to access your tokens stored on your BitPie wallet is to
```
Comment (kostko) [inline docs/general/manage-tokens/faq.md]: Wrap the line.

File: docs/general/manage-tokens/faq.md [hunk 2]
```diff
@@ -67,17 +62,25 @@ which you can
...
 to convert your BitPie mnemonic to the Oasis private key using the [Oasis
-unmnemonic CLI tool].
+unmnemonic CLI tool] (64-bit binaries available for [Linux][unmnemonic-linux],
+[MacOS][unmnemonic-macos] and [Windows][unmnemonic-windows]).
+
+Open a terminal (on Windows run `cmd`), move to the corresponding folder and
+execute it. An interactive prompt will be shown to you:
+
+1. select the **Bitpie** algorithm,
+2. enter the number of mnemonics (typically **12**) and carefully type them in
```
Comment (kostko) [inline docs/general/manage-tokens/faq.md]:
```suggestion
2. enter the number of words in the mnemonic (typically **12**) and carefully type them in
```

File: docs/general/manage-tokens/faq.md [hunk 3]
```diff
@@ -150,42 +161,126 @@ Here's a task for you:
...
+? Output directory /home/oa/tmp/wallet-export-2023-11-28
+ Index[4]: oasis1qqwkm23pyg638xvl2nu00frxhapusjjv8qhh3p77.private.pem - done
+Done writing wallet keys to disk, goodbye.
+```
+
+Move to the output directory. There you should see a `.private.pem` file
+containing the private key and named after the address it belongs to. Open it
+with a text editor and copy the Base64-encoded key between the
+`-----BEGIN ED25519 PRIVATE KEY-----` and `-----END ED25519 PRIVATE KEY-----`,
+and [paste it into the Oasis wallet][oasis-wallet-import-private-key].
```
Comment (kostko) [inline docs/general/manage-tokens/faq.md]: Couldn't one use the new `import-file` to do the import directly?

Comment (matevz) [reply docs/general/manage-tokens/faq.md]: Great idea! Added.

**Review by kostko — APPROVED** [review]

File: docs/general/manage-tokens/how-to-transfer-rose-into-paratime.mdx [hunk 4]
```diff
@@ -4,22 +4,24 @@ import {findSidebarItem} from '@site/src/sidebarUtils';
 # How to Transfer ROSE into a ParaTime
 
 This guide will walk you through transferring ROSE tokens from an Oasis
-consensus layer account into an Emerald, Sapphire or Cipher ParaTime using the
+consensus layer account into a Sapphire, Emerald or Cipher ParaTime using the
```
Comment (aefhm) [inline docs/general/manage-tokens/how-to-transfer-rose-into-paratime.mdx:7]: I see we use "into" as opposed to "to" or "on to" which I slightly prefer (purely pedantically).🤔

**Pull request merged**